npm-package/config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "mainnet": {
+    "url": "http://10.200.10.1:8545",
+    "chainId": 8995,
+    "coreApi": "0xf3E5A65851C3779f468c9EcB32E6f25D9D68601a",
+    "token": "0xbAA81A0179015bE47Ad439566374F2Bae098686F",
+    "oldToken": "0x6d0F3bF9aD2455b4F62f22fFD21317e1E3eEFE5C",
+    "otherToken": "0x642D2B84A32A9A92FEc78CeAA9488388b3704898",
+    "nodeRegistry": "0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF",
+    "uniswapAdapter": "0xE4eA76e830a659282368cA2e7E4d18C4AE52D8B3",
+    "uniswapFactory": "0xd2D23b73A67208a90CBfEE1381415329954f54E2",
+    "uniswap2Adapter": "0x0bADa0acE1d16ABf1ce1aAED9Bc7Ce231ECc35b5",
+    "uniswap2Router": "0xeE1bC9a7BFF1fFD913f4c97B6177D47E804E1920",
+    "ens": "0x92E8435EB56fD01BF4C79B66d47AC1A94338BB03",
+    "fifs": "0x57B81a9442805f88c4617B506206531e72d96290",
+    "ensResolver": "0xBc0c81a318D57ae54dA28DE69184A9c3aE9a1e1c",
+    "dataUnionFactory": "0x4bbcBeFBEC587f6C4AF9AF9B48847caEa1Fe81dA",
+    "dataUnionTemplate": "0x7bFBAe10AE5b5eF45e2aC396E0E605F6658eF3Bc",
+    "tokenMigrator": "0x352328769A92EfD179c6f61b57778868Bb3Ac13b",
+    "tokenMediator": "0xedD2aa644a6843F2e5133Fe3d6BD3F4080d97D9F",
+    "amb": "0xaFA0dc5Ad21796C9106a36D68f69aAD69994BB64",
+    "omnibridge": ""
+  },
+  "xdai": {
+    "url": "http://10.200.10.1:8546",
+    "chainId": 8997,
+    "streamRegistry": "0xa86863053cECFD9f6f861e0Fd39a042238411b75",
+    "storageNodeRegistry": "0xbAA81A0179015bE47Ad439566374F2Bae098686F",
+    "streamStorageRegistry": "0xE4eA76e830a659282368cA2e7E4d18C4AE52D8B3",
+    "binanceAdapter": "0xdc5F6368cd31330adC259386e78604a5E29E9415",
+    "uniswap2Router": "0xd2D23b73A67208a90CBfEE1381415329954f54E2",
+    "ensCache": "0xD1d514082ED630687a5DCB85406130eD0745fA06",
+    "chainlinkOracle": "0xD94D41F23F1D42C51Ab61685e5617BBC858e5871",
+    "linkToken": "0x3387F44140ea19100232873a5aAf9E46608c791E",
+    "dataUnionFactory": "0x4A4c4759eb3b7ABee079f832850cD3D0dC48D927",
+    "dataUnionTemplate": "0x36afc8c9283CC866b8EB6a61C6e6862a83cd6ee8",
+    "tokenMigrator": "0x1deF1497DF0e103D58Fd14C4f8e0365fe5F94423",
+    "token": "0x73Be21733CC5D08e1a14Ea9a399fb27DB3BEf8fF",
+    "tokenMediator": "0xedD2aa644a6843F2e5133Fe3d6BD3F4080d97D9F",
+    "amb": "0xaFA0dc5Ad21796C9106a36D68f69aAD69994BB64",
+    "omnibridge": ""
```

Comment:
omnibridge addresses should probably be found from inside the dockers that deploy them